_includes/manuals/nightly/5.2.4_cr_gce.md

```diff
@@ -1,40 +1,48 @@
-* Requires client e-mail address of an authorised Google Cloud Console client ID is entered in the new compute resource screen and its associated .p12 private key file is manually transferred to the foreman server.
+* Requires client e-mail address of an authorised Google Cloud Console client ID is entered in the new compute resource screen and its associated .json private key file is manually transferred to the foreman server.
 * The certificate must be stored in a location the foreman user account has permission to read.
-* If your server enforces SELinux ensure the context is suitable or relabel it using `restorecon -vv /usr/share/foreman/gce.p12`
-* Specify the location *on the foreman server* as the certificate path value e.g /usr/share/foreman/gce.p12
-* Ensure images are associated under the *Images* tab on the compute resource.
-* Add a provisioning template of type *finish* which will be executed over SSH on the new image.
-* Ensure the finish template is associated to the OS (on the *Associations* tab) and is set as the default for the operating system too.
+* If your server enforces SELinux ensure the context is suitable or relabel it using `restorecon -vv /usr/share/foreman/gce.json`
+* Specify the location *on the foreman server* as the certificate path value e.g /usr/share/foreman/gce.json
+* Ensure images are associated under the *Images* tab on the compute resource
+	* Ensure the correct username except *root* is set for Foreman to SSH into the image (if using SSH provisioning). Note that - For security reasons, Google do not provide the ability to ssh in directly as root.
```

Comment:
not sure if it's intentional but some of the dots seem to be indented?

Reply:
Yes, to keep description format consistent with other CR, I have added it intentionally.
These sub-points are related to image so placed it under *Images* Tab.